Dokumentationen/test/test_cases.adoc

```diff
@@ -315,13 +315,17 @@ a| * gefilterte, inkl. aller gebuchten Experimente, Dozentenwoche
 
 |===
 
-|*Beschreibung* |
+|*Beschreibung* |Es muss sichergestellt werden, dass der Admin keinen Zugriff auf das Dozenten-Journal hat, die Dozenten nur ihr persönliches Journal einsehen dürfen und nur der Admin die Statistik abrufen kann. Auch dürfen die Dozenten nur ihre eigene Dozentenwoche abrufen. 
```

Comment:
Evtl. nicht in der "muss" Form, weil das nach einer Anforderung klingt. "Es wird erwartet" wurde bis jetzt bei den restlichen TCs verwendet.